arrows/ffmpeg/ffmpeg_video_output.cxx

```diff
@@ -397,7 +397,7 @@ ::implementation_settings() const
   result->frame_rate = d->video->video_stream->avg_frame_rate;
   avcodec_parameters_from_context( result->parameters.get(),
                                    d->video->codec_context.get() );
-  result->klv_stream_count = 0; // TODO
+  result->klv_streams = {}; // TODO
```

Comment:
Do we know why there is a TODO comment here with no context? Can we just remove it?